portal/views/healthcheck.py

```diff
@@ -39,30 +42,29 @@ def celery_available():
     if result:
         return True, 'Celery is available.'
     else:
-        return False, 'Celery is not available'
+        return False, 'Celery is not available.'
 
 
 def celery_beat_available():
     """Determines whether celery beat is available"""
-    rs = redis.from_url(current_app.config['REDIS_URL'])
-
-    # When celery beat is running it pings
-    # our service periodically which sets
-    # 'celery_beat_available' in redis. If
-    # that variable expires it means
-    # we have not received a ping from celery beat
-    # within the allowed window and we must assume
-    # celery beat is not available
-    last_celery_beat_ping = rs.get('last_celery_beat_ping')
-    if last_celery_beat_ping:
-        return (
-            True,
-            'Celery beat is available. Last check: {}'.format(
-                last_celery_beat_ping
-            )
-        )
+    # https://github.com/celery/celery/issues/3694
+    # HealthCheck from https://goo.gl/tgbLeC
+
+    now = datetime.now(tz=pytz.utc)
+
+    # Name of the file used by PersistentScheduler to store
+    # the last run times of periodic tasks.
+    file_data = shelve.open('celerybeat-schedule')
```

Comment:
Does this require the web service and celery_beat to be running on the same machine or docker container?